packages/react/src/components/CircularProgressAvatar/CircularProgressAvatar.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import CircularProgressAvatar from './CircularProgressAvatar.tsx';
+
+export const meta = {
+  component: CircularProgressAvatar,
+  title: StoryConfig.CircularProgressAvatar.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <CircularProgressAvatar {...args} />;
+
+# Circular Progress Avatar
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+The Avatar with circular progress and badge.
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{color: 'warning',
+    progress: 70,
+    badgeOptions: {color: 'primary', badgeContent: '70%'}, 
+    avatarOptions: {
+      src: '/assets/images/avatar-john.svg', 
+      alt:'avatar', 
+      }
+    }}
```

Comment:
Can we move colors to the next line to make it more readable?

Reply:
Ack. Will send a seperate PR for this improvement.